assets/templates/stat.html

```diff
@@ -1,12 +1,13 @@
-<div class="column-two-thirds">
+<div class="column-three-quarters">
+  <h2 class="centre_name">{{name}}</h2>
   <table>
     <thead>
       <tr>
-        <th class="centre_name" scope="col">{{name}}</th>
+        <th scope="col">&nbsp;</th>
```

Comment:
Originally, the centre name was displayed inside this column. This has been moved outside of the table. The way the original design looked, I couldn't see a way of keeping the layout as a table, without introducing extra columns purely for presentational reasons, which isn't warranted.

For now, this column can be merged with the next one. Will be keen to discuss this layout with Tom.